src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Encapsulates the user interface for Duke.
+ *
+ * @author Hanif Kamal
+ */
+public class Ui {
+    private Scanner scanner;
+
+    /**
+     * Class constructor.
+     */
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Prints a welcome message.
+     */
+    public void welcome() {
```

Comment:
Should this be printWelcomeMessage?

Reply:
Yes, it should! Thank you very much for spotting this error. I have endeavoured to fix my mistake.